packages/cli/src/ceramic-daemon.ts

```diff
@@ -799,6 +804,32 @@ export class CeramicDaemon {
     })
   }
 
+  async validateAdminRequest(req: Request, res: Response, next: NextFunction): Promise<void> {
```

Comment:
this approach of separating the authorization logic from the actual core request logic is really nice.  Can we do it for the other admin API GET requests also?  Would also let us delete a lot of now-duplicated code

Reply:
yeah! commented above